frontend/public/components/_horizontal-nav.scss

```diff
@@ -1,7 +1,7 @@
 $co-m-horizontal-nav__menu-item-link-padding-lr: $pf-global-gutter;
 
 .co-m-horizontal-nav__menu {
-  border-bottom: 1px solid $color-grey-background-border;
+  border-bottom: 1px solid var(--pf-global--BorderColor--100);
```

Comment:
wondering if there is a list of css vars that need to be switch …eg:
`$color-grey-background-border`  - for -> `var(--pf-global--BorderColor--100)` ?

Reply:
I've included a find and replace for `$color-grey-background-border` in this PR https://github.com/openshift/console/pull/11136